test/unit/controllers/SubscriptionTestHelperSugar.scala

```diff
@@ -18,7 +18,7 @@ package unit.controllers
 
 import controllers.APISubscriptions
 import domain.APIStatus._
-import domain.{APIAccess, APISubscriptionStatus, APIVersion}
+import domain.{APIAccess, APIStatus, APISubscriptionStatus, APIVersion}
 import uk.gov.hmrc.play.test.UnitSpec
```

Comment:
Why is this in there if there are no changes in this file otherwise?

Reply:
That is the import for the trait which is used as part of method signature and therefore need to be there.